packages/blockchain-wallet-v4/src/redux/data/sfox/reducers.js

```diff
@@ -87,6 +87,9 @@ const sfoxReducer = (state = INITIAL_STATE, action) => {
     case AT.GET_BANK_ACCOUNTS_FAILURE: {
       return assoc('bankAccounts', Remote.Failure(payload), state)
     }
+    case AT.WIPE_BANK_ACCOUNTS: {
+      return assoc('bankAccounts', null, state)
```

Comment:
You should use Remote.NotAsked instead of null

Reply:
Good call, fixed in latest commit